README.md

```diff
@@ -2,13 +2,12 @@
 ___
 ## Platform (core)
 
-[TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER) | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS) | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2) | [SONAQUBE DASHBOARD](https://sonar.openmrs.org/dashboard?id=org.openmrs%3Aopenmrs)
------------- | ------------- | ------------- | -------------
-![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/TRUNK-MASTER) | ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/RESTWS-RESTWS) | ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/FHIR-FM2) | ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/SON-OPENMRSCOREMASTER)
+| [TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER)                                         | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS)                                      | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2)                                      | [SONAQUBE DASHBOARD](https://sonar.openmrs.org/dashboard?id=org.openmrs%3Aopenmrs)                   |
+|---------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
```

Comment:
@pasindur99 is this change part of the ticket requirement?

Reply:
@kdaud The changes are auto-generated through my IDE. I will replace it and update the pr.

Reply:
@kdaud Fixed this with [3b7d891](https://github.com/openmrs/openmrs-contrib-qaframework/pull/429/commits/3b7d8910bd19b5ab1f56c66467d8ce965cdae305)